subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/DefaultTransformer.java

```diff
@@ -141,33 +142,30 @@ public int hashCode() {
     }
 
     private static class TransformServiceLookup implements ServiceLookup {
-        private final File inputFile;
-        private final File outputDir;
-        private final Object parameters;
-        private final ArtifactTransformDependencies artifactTransformDependencies;
+        private final ImmutableList<InjectionPoint> injectionPoints;
```

Comment:
I like the name. We could use `@InjectionPointAnnotation` as the name for the meta-annotation, too.

Reply:
I added the meta annotation.